<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务器主窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;
/* 全局样式文件表 */
QWidget, QMainWindow, QLabel, QDialog, QTableView {
    font: 9pt &quot;Microsoft YaHei UI&quot;;
}

/* === QPushButton === */
QLineEdit, QPushButton {
    border: 1px solid #333333;
    padding: 3px;
    min-width: 65px;
    min-height: 12px;
}

QPushButton:hover {
    background-color: lightslategrey;
    border-color: #444444;
}

QPushButton:pressed {
    background-color: #444444;
    border-color: #333333;
    color: antiquewhite;
}

QPushButton:disabled {
    color: #333333;
}

/* 表格 */
QTableView, QTableWidget {
    border: 1px solid gray;
    background: white;
    gridline-color: gray;
}

QTableView::item, QTableWidget::item {
    padding-left: 5px;
    padding-right: 5px;
    border: none;
    background: lightgrey;
    border-right: 1px solid gray;
    border-bottom: 1px solid gray;
}

QTableView::item:selected, QTableWidget::item:selected {
    background: aquamarine;
    color: #444444;
}

QTableView::item:selected:! active, QTableWidget::item:selected:! active {
    color: black;
}

QTableView::indicator, QTableWidget::indicator {
    width: 20px;
    height: 20px;
}

QHeaderView::section {
    background: qlineargradient(spread:pad,
    x1: 0,
    y1: 0,
    x2: 0,
    y2: 1,
    stop: 0 rgba(160, 160, 160, 255),
    stop: 1 rgba(140, 140, 140, 255));
    color: #333333;
    border: 1px solid gray;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>服务器IP地址：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>服务器端口号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>开启监听</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>文件保存路径：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>文件保存路径</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>当前客户数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>连接数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>服务器状态</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>客户端连接状态</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>客户端IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>客户ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>客户地址</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>客户进出flag</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>图片接收状态</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_2">
          <column>
           <property name="text">
            <string>客户端IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>文件名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>图片大小</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>下载进度</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>保存位置</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>是否识别</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>是否校验</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
